README.md

```diff
@@ -1,9 +1,6 @@
 # Create Blazor DateTimePicker Component in Blazor Server App
 
-A quick start blazor server app that allows you to configure the Blazor DateTimePicker component of Syncfusion with its available options. This project contains simple code customizations, as well as some important features such as setting specific date time value, formatting and range restriction for date time selection.
-Documentation: https://blazor.syncfusion.com/documentation/datetime-picker/getting-started
-
-Online examples: https://blazor.syncfusion.com/demos/datetime-picker/default-functionalities
+[Blazor DateTime Picker](https://www.syncfusion.com/blazor-components/blazor-datetime-picker?utm_source=github&utm_medium=listing&utm_campaign=blazor-calendars-github-samples) is a lightweight and mobile-friendly component that allows end users to enter or select date and time values from a pop-up calendar and drop-down time list. A quick start blazor server app that allows you to configure the Blazor DateTimePicker component of Syncfusion with its available options. This project contains simple code customizations, as well as some important features such as setting specific date time value, formatting and range restriction for date time selection.
```

Comment:
Is this correct?
![image](https://user-images.githubusercontent.com/61148953/176998369-356262e3-d2a7-454d-b285-8762f4f4d880.png)
![image](https://user-images.githubusercontent.com/61148953/176998383-c30a454f-c485-46bc-be44-6fc109d2e290.png)

Check this in all places